templates/class-overview.html

```diff
@@ -23,9 +23,9 @@ <h2>{{auth.class_overview}}: {{class_info.name}}</h2>
             <td>{{student.last_login}}</td>
             <td>{{student.highest_level}}</td>
             <td>{{student.programs}}</td>
-            <td><a href="{{localize_link ('/programs?user=' + student.username)}}">{{auth.programs}}</a></td>
+            <td><a href="{{localize_link ('/programs?user=' + student.username)}}">{{student.username}}'s {{auth.page}}</a></td>
             <td>{% if student.latest_shared %}<a target="_blank" href="{{student.latest_shared.link}}">{{student.latest_shared.name}} ({{student.latest_shared.level}})</a>{% else %}-{% endif %}</td>
-            <td><a href="#" onclick="hedyApp.remove_student ('{{class_info.id}}', '{{student.username}}')">{{auth.remove_student}}</a></td>
+            <td><a href="#" onclick="hedyApp.remove_student ('{{class_info.id}}', '{{student.username}}')">{{student.username}} {{auth.remove}}</a></td>
```

Comment:
haha this is an interesting localization issue! For Dutch "Hedy verwijderen" makes sense but in En you would het "Hedy remove" which is weird and I do not know the proper order for other langs. I guess swapping the order and making it into _gebiedende wijs_: "remove Hedy/verwijder Hedy" is better?

Reply:
Should be fixed with the last commit!